.gitignore

```diff
@@ -3,6 +3,7 @@
 /*.yaml
 /*.yml
 *.exe
+.idea
```

Comment:
this is better fit in your global gitignore file

Reply:
you are right, fixed